spec/models/concerns/schema_validations_spec.rb

```diff
@@ -50,13 +50,29 @@
     end
   end
 
-  describe 'unique index valdiations' do
+  describe 'unique index validations' do
     subject { user_status.errors.full_messages }
 
     let(:user_status) { create :user_status }
 
     before { user_status.update(name: UserStatus.first.name) }
 
     it { is_expected.to include 'Name has already been taken' }
+
+    context 'included in skipped constant' do
+      let(:division) { create :division, division_status: 'status_pending' }
+      let(:division2) { create :division, division_status: 'status_pending' }
+      let(:division3) { create :division, division_status: 'status_active' }
+
+      before do
+        division2.update(name: division.name)
+        division3.update(name: division.name)
+      end
+
+      it 'does not add validation based on schema' do
```

Comment:
this is a very specific test, which fails if index is not skipped because the where clause in the index would not run otherwise causing the division to be invalid